src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/ServiceValueResolver.php

```diff
@@ -53,6 +55,8 @@ public function resolve(Request $request, ArgumentMetadata $argument)
             $controller = $controller[0].'::'.$controller[1];
         }
 
+        $controller = ltrim($controller, '\\');
```

Comment:
consistently with the previous comment, wrap this call in `if ('\\' === $controller[0]) {...}`

Reply:
Ok, thanks Nicolas.